NextcloudClient/NextcloudClient.cs

```diff
@@ -1565,7 +1565,7 @@ public NextcloudClient(string url, string userId, string password)
         /// <param name="path">remote Path.</param>
         private Uri GetDavUri(string path)
         {
-            return new Uri(_url + "/" + Davpath + path);
+            return new Uri(_url + "/" + Davpath + Uri.EscapeDataString(path).Replace("%2F", "/"));
```

Comment:
Ah, I can see the problem. Thanks for the hint, let me think about it. Your solution looks good do far, but maybe there are more place where this must be changed.